examples.md

```diff
@@ -168,8 +168,8 @@ We cache the elements of the Cabal store separately, as the entirety of `~/.caba
       ~/.cabal/packages
       ~/.cabal/store
       dist-newstyle
-    key: ${{ runner.os }}-${{ matrix.ghc }}
-```
+    key: cabal-${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('**/*.cabal', '**/cabal.project', '**/cabal.project.freeze') }}
```

Comment:
Remove `cabal-` from the beginning for key and restore-keys as it is unnecessary and making the key long. Also move 3 backticks for markdown to last line not just the end of code.

Reply:
![image](https://user-images.githubusercontent.com/21038781/147725732-38874ced-5b4b-4d8f-9c7c-43e89060bf6d.png)
Also, you can see the rich diff at https://github.com/actions/cache/pull/706/files?short_path=cfb2db6#diff-cfb2db6f153a4484d9bbd408f3227a264dce68fadd747cecdc30bda012b4d8cd . You have added the change under java-gradle instead of haskell-cabal.

Reply:
Sorry, maybe I shoudn’t have used the Github editor